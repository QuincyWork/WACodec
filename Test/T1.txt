deLWCaqEQW6PIQ9tcgqkvL2qkLAelsOlHsjLDTibdlLss3sLyxOuLXTiLMxkvfdJsogPSmLYZijttL6AkvBdL8nfX4uuohLIjVivT5uQQMhkLuDpLGyFkb1bvsvlus1drPyIkHUij0gPImsQOCsLaReLsIRSeK6llbj3usP3Sif)uL0qPuILIsLNsmvkUQskFvrQCDf2PqZujvU2IezUuk1LxKK9c)LenyskhwWIPu9ys1KrXLfTzPQplvgTs0Pv1QrPeVwsXSL42sYUvXVPQHtLoUIe1Yr1ZLY0ro9IKA7kQ(UsY4jP6wvkPHgmqcWacsdYg7DZEBAqQGWaHbgimFJTmo60ppHekfiP6UHdtYa7q4z37pghUjKoUF4cPLLEBX)hcs9RgZ1AxNIozG8mqM6PHTN(P1wNk2x2hiNrLH)DDjVHOgKIpWadekuYdTbdKQrHEyabccfk5HGbs1OqpmGabrVeAnyG0(txjHSOI1En6S1IgfzhiiJwQuVeAnieFKq4(scdKQrHEyabcsFqNE)bgivJc9Wacee35UuOKhcgivJc9WaceeEqpHbs1OqpmGabH577F9rHCadKQrHEyabceKGo9Ztd)76sETEMeXlAwGGOddIAWaXX1((wsiEzZYcIDpNj5SXx8Cyqudrle3q0crviAH4nIAGG07pKtLqxdY11mRBbrfHypWzJV45WGOgIwS3eTjSUNyTpB2MQDviUbNyDVnJfRjtSuPAMQjSuH(l73iQcrl2BI2ew3tS2NnBt1Ukqqk(admqOqjp0gmqQgf6HbeiiuOKhcgivJc9Wacee9sO1Gbs7pDLeYIkw71OZwlAuKDGGmAPs9sO1qDi8GEcdKQrHEyabcI7CxkuYdbdKQrHEyabcsFqNE)bgivJc9WaceeUVKWaPAuOhgqGGW899V(OqoGbs1OqpmGabccp7E)X4WnHeg8G9ahsJc610GDi)5jL6(k3ssjdIAqOuDAWtiRNx55oGy3Zzs(1((wsiQPbXUNZK8fCAjH4ei29CMKxpOxdd37cPE5DUZdCgfuBXpu8DkOw9GEnDC)4WOGAlg41ajOt)80W)UUKxRNjr8IMfeMV5wcomP606YEcIcQPGAqi(31LCyGqoCtinkqJgKXjNNhcsyqEiU(vjxP3n5bYZHWwhYOLkD9RsoQdjNNhcIT4xLCiJwQCCEc1H46xLCYHBIO2eiU(vjhIQniJwQ01Vk5Kd3e1HOpAKVs5YNjHqFvYaccLQtWoKL53TKqCZcc9vjh2HypOxdd37YgFXZHbXBe1ab5mQm8VRl5ne1GypOxth3pomSXx8CyqudoXAYmTz2OTzPANLnSMXc9x2VrCdoXAIM22nRDlv7Q22T0q)L9BevbNyX62ILL2mlv32SRsBc6VSFJ4nIwS3KDqqw9m0se3yzbXUNZKCuhIDpNj5Y78ernipdKPEAy7PFAT1BimFZTeCys1j29DljikOgeMVXwghD6NNqcLcKx3FGDi)5jLnkWPKbrnqiUbdehx77BjH4Lnlli29CMKZgFXZHbrneTqCdrlevHOfI3iQbcsV)qovcDnixxZSUfeveI9aNn(INddIAiAXE2SPTZAgRTj37w2mdXn40U1mw3AZUT2ynzY2(o6VSVJOk4eRjZ0MzJ2MLQDw2WAgl0FzFheKIpWadekuYdTbdKQrHEyabccfk5HGbs1OqpmGabrVeAnyG0(txjHSOI1En6S1IgfzhiiJwQuVeAnuhc3xsyGunk0ddiqqCN7sHsEiyGunk0ddiqq6d607pWaPAuOhgqGGWd6jmqQgf6HbeiimFF)RpkKdyGunk0ddiqGGWZU3FmoCtiHbpypWH0OGEnnyhYFEsPUVYTKuYGOgekvNg8eY65vEUdi29CMKFTVVLeIAAqS75mjFbNwsiobIDpNj51d61WW9UqQxEN78aNrb1w8dfFNcQvpOxth3pomkO2IbEnqc60ppn8VRl516zseVOzbH4FxxYHbc5WnH0OanAqgNCEEiiHb5H46xLCLE3KhiphYcVqGmAPsx)QKJ6qY55HGyl(vjhYOLkhNNqDiU(vjNC4MiQnbIRFvYHOAdYOLkD9Rso5WnrDi6Jg5RuU8zsi0xLmGGqP6eSd519hyhYY87wsiUzbH(QKd7qSh0RHH7DzJV45WG4nIAGGCgvg(31L8gIAqy(MBj4WKQtRl7jikOMcQbz1ZqlrCJLfe7EotYrDi29CMKlVZte1G8mqM6PHTN(P1wvbH5BULGdtQoXUVBjbrb1GW8n2Y4Ot)8esOuGypOxth3pomSXx8CyqudoXAYmTz2OTzPANLnSMXc9x2VrCdoXAIM22nRDlv7Q22T0q)L9BevbNyX62ILL2mlv32SRsBc6VSFJ4nIwS3K979T7z3A2yDZ6geK)8KYgf4uYGOgievbdehx77BjH4Lnlli29CMKZgFXZHbrneTqCdrlevHOfI3iQbcsV)qovcDnixxZSUfeveI9aNn(INddIAWPz33Bw3AS2VVVvTRTJ(l7QqCdonBxBJ1mwQ2VVNznXYc9x2vHOk4elw3wSS0MzP62MDvAtq)L9Bqqk(admqOqjp0gmqQgf6HbeiiuOKhcgivJc9Wacee9sO1Gbs7pDLeYIkw71OZwlAuKDGGmAPs9sO1qDi8GEcdKQrHEyabcI7CxkuYdbdKQrHEyabcsFqNE)bgivJc9WaceeUVKWaPAuOhgqGGW899V(OqoGbs1OqpmGabccp7E)X4WnHeg8G9ahsJc610GDi)5jL6(k3ssjdIAqOuDAWtiRNx55oGy3Zzs(1((wsiQPbXUNZK8fCAjH4ei29CMKxpOxdd37cPE5DUZdCgfuBXpu8DkOw9GEnDC)4WOGAlg41ajOt)80W)UUKxRNjr8IMfeMV5wcomP606YEcIcQPGAqi(31LCyGqoCtinkqJgKXjNNhcsyqEiU(vjxP3n5bYZHSWleiJwQ01Vk5OoKCEEii2IFvYHmAPYX5juhIRFvYjhUjIAtG46xLCiS2GmAPsx)QKtoCtuhI(Or(kLlFMec9vjdiiuQob7qOVk5WoKL53TKqCZcI9GEnmCVlB8fphgeVrudeKZOYW)UUK3qudI9GEnDC)4WWgFXZHbrn4eRjZ0MzJ2MLQDw2WAgl0Fz)gXn4eRjAAB3S2TuTRAB3sd9x2VrufCIfRBlwwAZSuDBZUkTjO)Y(nI3iAXEt2bbz1ZqlrCJLfe7EotYrDi29CMKlVZte1G8mqM6PHTN(P1w3GW8n3sWHjvNy33TKGOGAqy(gBzC0PFEcjukqED)b2H8NNu2OaNsge1aH4nmqCCTVVLeIx2SSGy3ZzsoB8fphge1q0cXneTqufIwiEJOgii9(d5uj01GCDnZ6wquri2dC24lEomiQbN2NXs1UMLMLkRzS2MXc9x2VrCdonHfRBBUTXM9Blv332CJ(l7QqufCAxfRz2yZKnRTzBSS02q)L9Bqqk(admqOqjp0gmqQgf6HbeiiuOKhcgivJc9Wacee9sO1Gbs7pDLeYIkw71OZwlAuKDGGmAPs9sO1qDiCFjHbs1OqpmGabXDUlfk5HGbs1OqpmGabPpOtV)adKQrHEyabccpONWaPAuOhgqGGW899V(OqoGbs1OqpmGabccp7E)X4WnHeg8G9ahsJc610GDi)5jL6(k3ssjdIAqOuDAWtiRNx55oGy3Zzs(1((wsiQPbXUNZK8fCAjH4ei29CMKxpOxdd37cPE5DUZdCgfuBXpu8DkOw9GEnDC)4WOGAlg41ajOt)80W)UUKxRNjr8IMfeI)DDjhgiKd3esJc0ObzCY55HGegKhIRFvYv6DtEG8Cil8cbYOLkD9RsoQdjNNhcIT4xLCiJwQCCEc1H46xLCYHBIO2eiU(vjhYSniJwQ01Vk5Kd3e1HOpAKVs5YNjHqFvYaccLQtWoe7b9A64(XHHn(INddIAWjwtMPnZgTnlv7SSH1mwO)Y(nIBWjwt002UzTBPAx12ULg6VSFJOk4elw3wSS0MzP62MDvAtq)L9BeVr0I9MSdcc9vjh2HSm)ULeIBwqSh0RHH7DzJV45WG4nIAGGCgvg(31L8gIAqy(MBj4WKQtRl7jikOMcQbz1ZqlrCJLfe7EotYrDi29CMKlVZte1G8mqM6PHTN(P1w1GW8n3sWHjvNy33TKGOGAqy(gBzC0PFEcjukqED)b2H8NNu2OaNsge1aH4omq69hsENNiQndcZ3ClbhMuDccBnx)QKRaBfioU233scXlBwwqc60ppn8VRl516zseTG8NNuQ7RCljLmiQbr9R99TKq8I2MnqwMF3scoTBXI1((Sjwt2TzIkTn0F5EceI)DDjhgiJwQ01Vk5Woex)QKtoCte1Maz0sLU(vjNC4MOoe9rJ8vkx(mje6RsgiKd3esJc0ObzCY55HGegKhIRFvYHybX1Vk5k9UjpqEoKfgsoppeeBXVk5qgTuPBY1Zd56xLCuhYOLkhNNqDiUjxppKRFvYHybXn565HC9RsUsVBYdKNdzHbb5zGm1tdBp9tRTovSVSpS)fA2QtLTUqbz1ZqlHt7Bwt2MTBJn73w7Q02q)LBwqC5(oE2A2cp1)27piAb5mQm8VRl5ne1G0YsVT4)dbzDk6KbYFEszJcCkzqudsXhyGbcfk5H2Gbs1OqpmGabHcL8qWaPAuOhgqGGOxcTgmqA)PRKqwuXAVgD2ArJISds1OqpmqC2ArJISd1HSOI1En6S1IgfzhQdceKrlvQxcTgQdH7ljmqQgf6HbeiiUZDPqjpemqQgf6Hbeii9bD69hyGunk0ddiqq4b9egivJc9WaceeMVV)1hfYbmqQgf6Hbeiqq4z37pghUjekvNaHilyG44AFFljeVSzzbr3x8Cyqudrle3q0crviAbcsqN(5PH)DDjVwptIOfKL53TKquLfK)8KsDFLBjPKbrniuQon4jKfCOK3h0tBVEgMRNMTNUmuZ6h8qZ21MmFW2MuDAWZMTLP8iWtLt3YNsiQFTVVLeIxMzb5zGmvtr2M9pfMskOMcQPbz1ZqlrChcX)UUKddKrlvoopH6qihUjKgfOrdY4KZZdbjmipe9rJ8vkx(mje6Rsgi6LqRbzrfR9A0zRfnkYoi588qqw78KcQXMLbEqNs(3OxVbz0sL6LqRH6qgTuzO8ZtuheeUNc07piAJfKEEOkfIhOaI3qoJkd)76sEdrni)5jLnkWPKbrnifFGbgiuOKhAdgivJc9WaceekuYdbdKQrHEyabcIEj0AWaP9NUsczrfR9A0zRfnkYoqqgTuPEj0AOoeUVKWaPAuOhgqGG4o3LcL8qWaPAuOhgqGG0h0P3FGbs1OqpmGabHh0tyGunk0ddiqqy(((xFuihWaPAuOhgqGabHNDV)yC4MqOuDAWtqiobgioU233scXlBwwq09fphge1q0cXneTqufIwGGe0PFEA4FxxYR1ZKiAbzz(Dljevzb5ppPu3x5wskzqudcLQtdEczbhk59b902RNH56Pz7Pld1S(bp0SDTjZhSTjvNg8SzBzkpc8u50T8PeI6x77BjH4fvwqEgit1uKTz)tHPKcQPGABqw9m0se3Hq8VRl5Waz0sLJZtOoeYHBcPrbA0Gmo588qqcdYdrF0iFLYLptcH(QKbIEj0AqwuXAVgD2ArJISdsoppeK1opPGASzzGh0PK)n61BqgTuPEj0AOoKrlvgk)8e1bbH7Pa9(dI2ybPNhQsH4bkG4nKZOYW)UUK3qudYFEszJcCkzqudsXhyGbcfk5H2Gbs1OqpmGabHcL8qWaPAuOhgqGGOxcTgmqA)PRKqwuXAVgD2ArJISdeKrlvQxcTgQdHh0tyGunk0ddiqqCN7sHsEiyGunk0ddiqq6d607pWaPAuOhgqGGW9LegivJc9WaceeMVV)1hfYbmqQgf6Hbeiqq4z37pghUjekvNg8eeIZGbI6x77BjH4eli6(INddIAiAH4gIwiQcrlqqCCTVVLeIx2SSGe0PFEA4FxxYR1ZKiAbzz(Dljevzb5ppPu3x5wskzqudcLQtdEczbhk59b902RNH56Pz7Pld1S(bp0SDTjZhSTjvNg8SzBzkpc8u50T8PeYZazQMISn7FkmLuqnfutfKvpdTeXDie)76somqgTu548eQdHC4MqAuGgniJtoppeKWG8q0hnYxPC5ZKqOVkzGOxcTgKfvS2RrNTw0Oi7GKZZdbzTZtkOgBwg4bDk5FJE9gKrlvQxcTgQdz0sLHYpprDqq4EkqV)GOnwqoJkd)76sEdrni)5jLnkWPKbrnifFGbgiuOKhAdgivJc9WaceekuYdbdKQrHEyabcIEj0AWaP9NUsczrfR9A0zRfnkYoqqgTuPEj0AOoeUVKWaPAuOhgqGG4o3LcL8qWaPAuOhgqGG0h0P3FGbs1OqpmGabHh0tyGunk0ddiqqy(((xFuihWaPAuOhgqGabHNDV)yC4MqOuDAWtqGGmhrT7nnqaa