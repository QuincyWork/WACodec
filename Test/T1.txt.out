{
["m"]="d",
["s"]="2.4.21",
["d"]={
["expanded"]=false,
["load"]={
["difficulty"]={
["multi"]={
},
},
["faction"]={
["multi"]={
},
},
["class"]={
["single"]="DEMONHUNTER",
},
["talent"]={
["multi"]={
},
},
["race"]={
["multi"]={
},
},
["use_class"]="true",
["role"]={
["multi"]={
},
},
["talent2"]={
["multi"]={
},
},
["pvptalent"]={
["multi"]={
},
},
},
["selfPoint"]="BOTTOMLEFT",
["id"]="集中值",
["activeTriggerMode"]=-10,
["numTriggers"]=1,
["disjunctive"]="all",
["regionType"]="group",
},
["v"]=1421,
["a"]={
},
["c"]={
[1]={
["yOffset"]=-200,
["width"]=260,
["id"]="集中4",
["backdropColor"]={
[4]=1,
},
["regionType"]="aurabar",
["backgroundColor"]={
[1]=0.37647058823529,
[2]=0.37254901960784,
[3]=0.36862745098039,
[4]=0.75,
},
["trigger"]={
["power_operator"]="<",
["custom_hide"]="timed",
["power"]="32",
["use_powertype"]=true,
["unevent"]="auto",
["powertype"]=17,
["use_power"]=true,
["event"]="Power",
["type"]="status",
["use_unit"]=true,
},
["fontFlags"]="OUTLINE",
["borderBackdrop"]="Blizzard Dialog Background Dark",
["init_started"]=1,
["borderOffset"]=11,
["borderSize"]=1,
["texture"]="Armory",
["disjunctive"]="all",
["displayTextRight"]=" ",
["icon"]=false,
["border"]=true,
["borderColor"]={
[1]=0,
[2]=0,
[3]=0,
[4]=1,
},
["borderInset"]=7,
["timer"]=false,
["text"]=false,
["numTriggers"]=1,
["load"]={
["difficulty"]={
["multi"]={
},
},
["faction"]={
["multi"]={
},
},
["class"]={
["single"]="DEMONHUNTER",
},
["talent"]={
["multi"]={
},
},
["race"]={
["multi"]={
},
},
["use_class"]=true,
["role"]={
["multi"]={
},
},
["talent2"]={
["multi"]={
},
},
["pvptalent"]={
["multi"]={
},
},
},
["activeTriggerMode"]=-10,
["init_completed"]=1,
["barColor"]={
[1]=0.71764705882353,
[2]=0.35686274509804,
[3]=0.71764705882353,
},
["stacks"]=false,
["height"]=20,
["displayTextLeft"]="  ",
},
[2]={
["yOffset"]=-200,
["width"]=260,
["id"]="集中3",
["backdropColor"]={
[4]=1,
},
["regionType"]="aurabar",
["border"]=true,
["trigger"]={
["power_operator"]=">=",
["custom_hide"]="timed",
["power"]="32",
["use_powertype"]=true,
["unevent"]="auto",
["powertype"]=17,
["use_power"]=true,
["event"]="Power",
["type"]="status",
["use_unit"]=true,
},
["fontFlags"]="OUTLINE",
["borderBackdrop"]="Blizzard Dialog Background Dark",
["init_started"]=1,
["borderOffset"]=11,
["backgroundColor"]={
[1]=0.37647058823529,
[2]=0.37254901960784,
[3]=0.36862745098039,
[4]=0.75452484190464,
},
["texture"]="Armory",
["disjunctive"]="all",
["displayTextRight"]=" ",
["icon"]=false,
["borderSize"]=1,
["borderColor"]={
[1]=0,
[2]=0,
[3]=0,
[4]=1,
},
["borderInset"]=7,
["timer"]=false,
["text"]=false,
["displayTextLeft"]="  ",
["load"]={
["difficulty"]={
["multi"]={
},
},
["faction"]={
["multi"]={
},
},
["class"]={
["single"]="DEMONHUNTER",
},
["talent"]={
["multi"]={
},
},
["race"]={
["multi"]={
},
},
["use_class"]=true,
["role"]={
["multi"]={
},
},
["talent2"]={
["multi"]={
},
},
["pvptalent"]={
["multi"]={
},
},
},
["numTriggers"]=1,
["stacks"]=false,
["init_completed"]=1,
["barColor"]={
[1]=0.92156862745098,
[2]=0.14117647058823,
[3]=0.18823529411765,
},
["height"]=20,
["activeTriggerMode"]=-10,
},
[3]={
["yOffset"]=-200,
["width"]=260,
["id"]="集中2",
["backdropColor"]={
[4]=1,
},
["regionType"]="aurabar",
["backgroundColor"]={
[1]=0.37647058823529,
[2]=0.37254901960784,
[3]=0.36862745098039,
[4]=0.75,
},
["trigger"]={
["power_operator"]=">=",
["custom_hide"]="timed",
["power"]="62",
["use_powertype"]=true,
["unevent"]="auto",
["powertype"]=17,
["use_power"]=true,
["event"]="Power",
["type"]="status",
["use_unit"]=true,
},
["fontFlags"]="OUTLINE",
["borderBackdrop"]="Blizzard Dialog Background Dark",
["init_started"]=1,
["borderOffset"]=11,
["borderSize"]=1,
["texture"]="Armory",
["disjunctive"]="all",
["displayTextRight"]=" ",
["icon"]=false,
["border"]=true,
["borderColor"]={
[1]=0,
[2]=0,
[3]=0,
[4]=1,
},
["borderInset"]=7,
["timer"]=false,
["text"]=false,
["numTriggers"]=1,
["load"]={
["difficulty"]={
["multi"]={
},
},
["faction"]={
["multi"]={
},
},
["class"]={
["single"]="DEMONHUNTER",
},
["talent"]={
["multi"]={
},
},
["race"]={
["multi"]={
},
},
["use_class"]=true,
["role"]={
["multi"]={
},
},
["talent2"]={
["multi"]={
},
},
["pvptalent"]={
["multi"]={
},
},
},
["activeTriggerMode"]=-10,
["init_completed"]=1,
["barColor"]={
[1]=0.93725490196078,
[2]=0.94509803921569,
[3]=0.36862745098039,
},
["stacks"]=false,
["height"]=20,
["displayTextLeft"]="  ",
},
[4]={
["yOffset"]=-200,
["width"]=260,
["id"]="集中1",
["backdropColor"]={
[4]=1,
},
["regionType"]="aurabar",
["backgroundColor"]={
[1]=0.37647058823529,
[2]=0.37254901960784,
[3]=0.36862745098039,
[4]=0.75,
},
["trigger"]={
["power_operator"]=">=",
["custom_hide"]="timed",
["power"]="82",
["use_powertype"]=true,
["unevent"]="auto",
["powertype"]=17,
["use_power"]=true,
["event"]="Power",
["type"]="status",
["use_unit"]=true,
},
["fontFlags"]="OUTLINE",
["borderBackdrop"]="Blizzard Dialog Background Dark",
["init_started"]=1,
["borderOffset"]=11,
["borderSize"]=1,
["texture"]="Armory",
["disjunctive"]="all",
["displayTextRight"]=" ",
["icon"]=false,
["border"]=true,
["borderColor"]={
[1]=0,
[2]=0,
[3]=0,
[4]=1,
},
["borderInset"]=7,
["timer"]=false,
["text"]=false,
["displayTextLeft"]="  ",
["load"]={
["difficulty"]={
["multi"]={
},
},
["faction"]={
["multi"]={
},
},
["class"]={
["single"]="DEMONHUNTER",
},
["talent"]={
["multi"]={
},
},
["race"]={
["multi"]={
},
},
["use_class"]=true,
["role"]={
["multi"]={
},
},
["talent2"]={
["multi"]={
},
},
["pvptalent"]={
["multi"]={
},
},
},
["numTriggers"]=1,
["stacks"]=false,
["init_completed"]=1,
["barColor"]={
[1]=0.32549019607843,
[2]=0.85098039215686,
[3]=0.29803921568628,
},
["height"]=20,
["activeTriggerMode"]=-10,
},
[5]={
["yOffset"]=-200,
["width"]=73.999984741211,
["displayText"]="%power\
",
["id"]="集中值数字",
["xOffset"]=-129,
["init_started"]=1,
["load"]={
["difficulty"]={
["multi"]={
},
},
["faction"]={
["multi"]={
},
},
["class"]={
["single"]="DEMONHUNTER",
["multi"]={
["HUNTER"]=true,
["DEMONHUNTER"]=true,
},
},
["talent"]={
["multi"]={
},
},
["race"]={
["multi"]={
},
},
["use_class"]=true,
["role"]={
["multi"]={
},
},
["talent2"]={
["multi"]={
},
},
["pvptalent"]={
["multi"]={
},
},
},
["regionType"]="text",
["selfPoint"]="LEFT",
["activeTriggerMode"]=0,
["init_completed"]=1,
["numTriggers"]=1,
["progressPrecision"]=0,
["trigger"]={
["power_operator"]=">",
["custom_hide"]="timed",
["power"]="0",
["use_powertype"]=true,
["unevent"]="auto",
["powertype"]=17,
["percentpower_operator"]=">",
["use_power"]=false,
["event"]="Power",
["type"]="status",
["use_percentpower"]=true,
["percentpower"]="0",
["use_unit"]=true,
},
["height"]=36.000064849854,
["fontSize"]=18,
},
[6]={
["yOffset"]=-200,
["width"]=5,
["init_started"]=1,
["color"]={
[1]=0,
[2]=0,
[3]=0,
},
["xOffset"]=-80,
["numTriggers"]=1,
["load"]={
["difficulty"]={
["multi"]={
},
},
["faction"]={
["multi"]={
},
},
["class"]={
["single"]="DEMONHUNTER",
},
["talent"]={
["multi"]={
},
},
["race"]={
["multi"]={
},
},
["use_class"]=true,
["role"]={
["multi"]={
},
},
["talent2"]={
["multi"]={
},
},
["pvptalent"]={
["multi"]={
},
},
},
["regionType"]="texture",
["frameStrata"]=4,
["activeTriggerMode"]=0,
["init_completed"]=1,
["rotation"]=90,
["trigger"]={
["use_alive"]=true,
["use_class"]=true,
["custom_hide"]="timed",
["event"]="Unit Characteristics",
["use_unit"]=true,
["class"]="DEMONHUNTER",
["type"]="status",
["unevent"]="auto",
},
["id"]="分段  1",
["height"]=30,
["texture"]="Interface\\AddOns\\WeakAuras\\Media\\Textures\\Square_White",
},
[7]={
["yOffset"]=-200,
["width"]=5,
["init_started"]=1,
["color"]={
[1]=0,
[2]=0,
[3]=0,
},
["xOffset"]=-30,
["numTriggers"]=1,
["load"]={
["difficulty"]={
["multi"]={
},
},
["faction"]={
["multi"]={
},
},
["class"]={
["single"]="DEMONHUNTER",
},
["talent"]={
["multi"]={
},
},
["race"]={
["multi"]={
},
},
["use_class"]=true,
["role"]={
["multi"]={
},
},
["talent2"]={
["multi"]={
},
},
["pvptalent"]={
["multi"]={
},
},
},
["regionType"]="texture",
["frameStrata"]=4,
["activeTriggerMode"]=0,
["init_completed"]=1,
["rotation"]=90,
["trigger"]={
["use_alive"]=true,
["use_class"]=true,
["custom_hide"]="timed",
["event"]="Unit Characteristics",
["use_unit"]=true,
["class"]="DEMONHUNTER",
["type"]="status",
["unevent"]="auto",
},
["id"]="分段  2",
["height"]=30,
["texture"]="Interface\\AddOns\\WeakAuras\\Media\\Textures\\Square_White",
},
[8]={
["yOffset"]=-200,
["width"]=5,
["init_started"]=1,
["color"]={
[1]=0,
[2]=0,
[3]=0,
},
["xOffset"]=70,
["load"]={
["difficulty"]={
["multi"]={
},
},
["faction"]={
["multi"]={
},
},
["class"]={
["single"]="DEMONHUNTER",
},
["talent"]={
["multi"]={
},
},
["race"]={
["multi"]={
},
},
["use_class"]=true,
["role"]={
["multi"]={
},
},
["talent2"]={
["multi"]={
},
},
["pvptalent"]={
["multi"]={
},
},
},
["regionType"]="texture",
["numTriggers"]=1,
["activeTriggerMode"]=0,
["init_completed"]=1,
["rotation"]=90,
["trigger"]={
["use_alive"]=true,
["use_class"]=true,
["custom_hide"]="timed",
["event"]="Unit Characteristics",
["use_unit"]=true,
["class"]="DEMONHUNTER",
["type"]="status",
["unevent"]="auto",
},
["id"]="分段  3",
["height"]=30,
["texture"]="Interface\\AddOns\\WeakAuras\\Media\\Textures\\Square_White",
},
},
}